tests/system/io/Main.agda

```diff
@@ -0,0 +1,17 @@
+{-# OPTIONS --guardedness #-}
+
+module Main where
+
+open import Data.String.Base
+open import Function.Base using (_$_)
+open import IO
+open import System.Exit using (exitSuccess)
+
+main : Main
+main = run $ do
+  hSetBuffering stdout noBuffering
+  forever $ do
```

Comment:
It was really not clear to me that this would terminate, that somehow `getLine` receiving an empty line would 'escape' the forever! Perhaps comment?

Reply:
It's the `"exit"` keyword that makes the function exit. But I'll add a comment, sure.